codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -193,9 +195,9 @@ private FlowState connectionState() {
         return state(connection.connectionStream());
     }
 
-    private static FlowState state(Http2Stream stream) {
+    private FlowState state(Http2Stream stream) {
```

Comment:
This is used in inner class (not that you introduced this).  Make package protected.